src/ads/native-ad/NativeAdContext.ts

```diff
@@ -21,6 +21,6 @@ import type GoogleMobileAdsNativeView from '../../specs/components/GoogleMobileA
 
 type NativeAdContextType = {
   nativeAd: NativeAd;
-  viewRef: RefObject<React.ElementRef<typeof GoogleMobileAdsNativeView>>;
+  viewRef: RefObject<React.ComponentRef<typeof GoogleMobileAdsNativeView> | null>;
```

Comment:
this line is problematic - the type change is potentially breaking, need to either handle differently or verify not breaking

not sure if types are included in react now but they are in react-native and I don't think I'm using the built-in types yet, I think I'm still using DefinitelyTyped ones

Reply:
@dylancom I can't see a way around this one - our internal logic actually uses it's possible null-ness, I think we just need to issue it as a lightly breaking change.

https://github.com/invertase/react-native-google-mobile-ads/blob/8a5b6ef2abf8b407010ab4cef130caef26b89563/src/ads/native-ad/NativeAsset.tsx#L47-L50

I'm going to re-push with the commit altered so it will trigger the correct breaking change release and notes - I can also *undo* that if you disagree, but you'll get to see what I'll put in there for a final approval pre-merge + release

what do you think?